micronaut-camunda-bpm-example/src/main/java/info/novatec/micronaut/camunda/bpm/example/JettyServerCustomizer.java

```diff
@@ -0,0 +1,55 @@
+package info.novatec.micronaut.camunda.bpm.example;
+
+import info.novatec.micronaut.camunda.bpm.example.rest.RestApp;
+import io.micronaut.context.event.BeanCreatedEvent;
+import io.micronaut.context.event.BeanCreatedEventListener;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.jetty.servlet.ServletHolder;
+import org.glassfish.jersey.servlet.ServletContainer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Singleton;
+
+@Singleton
+public class JettyServerCustomizer implements BeanCreatedEventListener<Server> {
+
+    private static final Logger log = LoggerFactory.getLogger(JettyServerCustomizer.class);
+
+    @Override
+    public Server onCreated(BeanCreatedEvent<Server> event) {
+
+        Server jettyServer = event.getBean();
+
+        ServletContextHandler context = (ServletContextHandler) jettyServer.getHandler();
```

Comment:
should we add a link to https://micronaut-projects.github.io/micronaut-servlet/1.0.x/guide/#jetty ?